app/eventPage/[id].tsx

```diff
@@ -0,0 +1,86 @@
+import { useLocalSearchParams } from "expo-router";
+import React from "react";
+import { Button, ScrollView, SizableText, Text, XStack, YStack } from "tamagui";
+
+import { IconSymbol } from "@/components/ui/IconSymbol";
+import { useEventById } from "@/hooks/use-event";
+import { useUser } from "@/hooks/use-user";
+
+export default function EventPage() {
+  const { id } = useLocalSearchParams();
+  const { data: event } = useEventById(Number(id));
```

Comment:
I would like to see this changed so that the `event` variable is properly represented in that it is received as plural. So something like:

```suggestion
  const { data: events } = useEventById(Number(id));
  const event = events?.docs[0];
```